### ISSUE TITLE
An addon must define a `name` property error in 0.1.14

### ISSUE THREAD

**Issue by rlivsey** [bustlelabs/mobiledoc-dom-renderer#7]
Just updated our Ember app to use 0.1.14 and it fails to start, with the error:

```
An addon must define a `name` property.
```

Took a while to track down to this addon, but it's [thrown in models/addon.js in ember-cli][1] because when the addon is loaded in `Addon.lookup` [the `addonModule` at this point][2] isn't correct and the addon definition is under `addonModule.default` and not `addonModule` as it should be.

I assume this is something to do with the CommonJS bundling which came in 0.1.14 as going back to 0.1.13 works fine as an addon again.

[1]: https://github.com/ember-cli/ember-cli/blob/33e260cdb86416382cb9a711599e59ca35c5192f/lib/models/addon.js#L112
[2]: https://github.com/ember-cli/ember-cli/blob/33e260cdb86416382cb9a711599e59ca35c5192f/lib/models/addon.js#L725

**Comment by gdub22** [comment]
+1 Was about to open an issue to say accessing the module with `.default` is awkward in commonjs.

**Comment by mixonic** [comment]
@rlivsey we just ran into this. Please pin to 0.1.13 for now! Painful for us to debug, I imagine for you as well. Basically, Ember-CLI wants to use `main` for customizing the location of `index.js`, however commonjs consumers want to use it for the main file of the packages. Ugh.

@gdub22 CommonJS is awkward in that it doesn't support default and named exports :-) I'm not inclined to change the `.default` thing, since it would diverge the output from ES modules.

**Comment by rlivsey** [comment]
@mixonic looks like you can customize the location with `ember-addon: { main: "..." }` in package.json according to [the source][1]:

```javascript
  addonMain = addon.pkg['ember-addon-main'] || 
                       (addon.pkg['ember-addon'] && addon.pkg['ember-addon'].main) || 
                        addon.pkg['main'] || 
                        'index.js';
```

[1]: https://github.com/ember-cli/ember-cli/blob/33e260cdb86416382cb9a711599e59ca35c5192f/lib/models/addon.js#L696

**Comment by mixonic** [comment]
Excellent, thanks @rlivsey. Fixed and released as `0.1.15`.
